operators/endpointmetrics/pkg/util/status.go

```diff
@@ -16,47 +16,47 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-type StatusConditionName string
+type ConditionReason string
 
 const (
-	DeployedStatus           StatusConditionName = "Deployed"
-	DisabledStatus           StatusConditionName = "Disabled"
-	DegradedStatus           StatusConditionName = "Degraded"
-	NotSupportedStatus       StatusConditionName = "NotSupported"
-	MaxStatusConditionsCount                     = 10
+	Deployed                 ConditionReason = "Deployed"
+	Disabled                 ConditionReason = "Disabled"
+	Degraded                 ConditionReason = "Degraded"
+	NotSupported             ConditionReason = "NotSupported"
```

Comment:
I didn't see the original PR,  but I still prefer the old names. It's a quite common pattern in Go to const values of a typed-string after the original type's name, i.e. `ConditionReasonDeployed` and so on. Particularly helpful when this lives in an `util` package.

Reply:
I agree. I could have wait.
I've changed it because I'm putting these status management files into their own `status` package in a subsequent PR...